app/src/main/java/com/losandroides/learn/domain/ItemMapper.kt

```diff
@@ -11,3 +12,8 @@ fun ItemsDTO.toDomain(): List<Item> {
 }
 
 private fun ItemDTO.toDomain() = Item(title)
```

Comment:
I think the best place for the DTO to domain mapper should be the data package, because the domain layer should not know about the implementation of the data. It's not a big problem, and maybe I'm not right, but I prefer to do it in that way.

Reply:
Good catch! I'll change it.